### DISCUSSION TITLE
Storage

### DISCUSSION THREAD

**Discussion by Solipsos** [akash-network/community#328] [General]
Hello,

is there an easy way to store data on Akash?

regards

**Comment by piber-dev** [comment]
hello @Solipsos,

storing data is very easy on Akash, but the primary goal of the network is compute not storage which makes it harder and more expensive compared to networks dedicated only to storage, albeit more flexible. 

without long explanations, you could easily store data by hosting an Ubuntu server with SSH access (which also works with FTP and clients such as FileZilla). 

as long as you use persistent storage, this is as safe as relying on a single deployment & provider for a deployment. how to scale this, i'm not sure.

if you want to make it easier, cheaper, and likely more reliable without custom solutions, look for networks such as Sia, Storj, or other storage networks.

**Reply by pcfreak30** [reply]
@Solipsos Sia community developer here, if you want decentralized object storage, you can join the discord at https://sia.tech/discord and ask questions!

**Comment by brewsterdrinkwater** [comment]
If you have any more specific questions, please feel free to join the Akash Network Discord to talk to many members of the community in more real time. 

There are also Akash Insiders and Akash Vanguards that can help you deploy.
